lang/rust/README.md

```diff
@@ -1,46 +1,5 @@
-<!---
-  Licensed to the Apache Software Foundation (ASF) under one
-  or more contributor license agreements.  See the NOTICE file
-  distributed with this work for additional information
-  regarding copyright ownership.  The ASF licenses this file
-  to you under the Apache License, Version 2.0 (the
-  "License"); you may not use this file except in compliance
-  with the License.  You may obtain a copy of the License at
+# Apache Avro Rust
 
-    http://www.apache.org/licenses/LICENSE-2.0
+The development of Avro Rust has been moved to [apache/avro-rs](https://github.com/apache/avro-rs). Please submit issues and PRs to the new repository. 
 
-  Unless required by applicable law or agreed to in writing,
-  software distributed under the License is distributed on an
-  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-  KIND, either express or implied.  See the License for the
-  specific language governing permissions and limitations
-  under the License.
--->
-
-# apache-avro
-
-Apache Avro Rust SDK
-
-[![Current Crates.io Version](https://img.shields.io/crates/v/apache_avro.svg)](https://crates.io/crates/apache-avro)
-[![Documentation](https://img.shields.io/badge/docs-latest-blue)](https://docs.rs/apache-avro/latest/apache_avro/)
-[![CI](https://github.com/apache/avro/actions/workflows/test-lang-rust-ci.yml/badge.svg)](https://github.com/apache/avro/actions/workflows/test-lang-rust-ci.yml)
-![Rust Version](https://img.shields.io/crates/msrv/apache_avro.svg?label=MSRV&color=red)
-![license](https://shields.io/badge/license-Apache--2.0-blue)
-
-# Subprojects
-
-## Avro
-
-See [avro/README.md](./avro/README.md)
-
-## Avro derive
-
-See [avro_derive/README.md](./avro_derive/README.md)
-
-## Avro test helper
-
-See [avro_test_helper/README.md](./avro_test_helper/README.md)
-
-## WebAssembly demo application
-
-See [wasm-demo/README.md](./wasm-demo/README.md)
+We apologize for the inconvenience and greatly appreciate your contributions!
```

Comment:
❤️

Reply:
just curious, why only rust version moved to its own repo?

Reply:
It's an experiment, if it ends up being better for sustained development, the other languages will likely be split out as well.